configure.ac

```diff
@@ -177,6 +177,17 @@ AS_IF([test x"$with_utempter" != xno],
       [AC_MSG_WARN([Unable to find libutempter; utmp entries will not be made.])],
       [AC_MSG_ERROR([--with-utempter was given but libutempter was not found.])])])])
 
+AC_ARG_ENABLE([syslog],
+  [AS_HELP_STRING([--enable-server], [Enable connection logging in mosh-server @<:@yes@:>@])],
```

Comment:
I think you mean `--enable-syslog` here.  Also, I think I want this to default to no for now.